src/include/duckdb/planner/filter/in_filter.hpp

```diff
@@ -19,12 +19,11 @@ class InFilter : public TableFilter {
 
 public:
 	explicit InFilter(vector<Value> values);
-	InFilter(vector<Value> values, bool origin_is_hash_join);
 
 	vector<Value> values;
-	bool origin_is_hash_join;
 
 public:
+	static optional_ptr<InFilter> ExtractFromOptional(TableFilter &filter);
```

Comment:
Can we remove this method? The `InFilter` is not specially related to the `OptionalFilter` so this method seems out-of-place